wiki/Tournaments/ASC/2/en.md

```diff
@@ -0,0 +1,208 @@
+---
+tags:
+  - ASC
+  - ASC 2
+  - Asia Suiji Cup
+---
+
+# Asia Suiji Cup 2: Autumn Rhythm
+
+![ASC 2 banner](img/asc-2-banner.jpg)
+
+The **Asia Suiji Cup 2: Autumn Rhythm** (***ASC 2***) is a three-on-three asian and oceanian countries restricted double-elimination osu!mania 4-key tournament hosted by ::{ flag=MY }:: [\[-Leon-\]](https://osu.ppy.sh/users/13382147), ::{ flag=VN }:: [MashedPotato](https://osu.ppy.sh/users/10494860), and ::{ flag=MY }:: [rantable](https://osu.ppy.sh/users/13630137). Each participant registered individually and is assigned to a random team. It is the second instalment of the Asia Suiji Cup, a successor, and inspired by the [Asian's Campionships](/wiki/Tournaments/o!mAC) and [SEA Champion](/wiki/Tournaments/SEAC) series .
```

Comment:
Mixing tenses here is a bit weird. Considering the team drawing phase is already over, we could fine-tune it. Also some other misc wording changes
```suggestion
The **Asia Suiji Cup 2: Autumn Rhythm** (***ASC 2***) is a three-on-three asian and oceanian countries restricted double-elimination osu!mania 4-key tournament hosted by ::{ flag=MY }:: [\[-Leon-\]](https://osu.ppy.sh/users/13382147), ::{ flag=VN }:: [MashedPotato](https://osu.ppy.sh/users/10494860), and ::{ flag=MY }:: [rantable](https://osu.ppy.sh/users/13630137). Each participant registered individually and was assigned to a random team. It is the second instalment of the Asia Suiji Cup, its successor, and inspired by the [Asian's Campionships](/wiki/Tournaments/o!mAC) and [SEA Champion](/wiki/Tournaments/SEAC) series.
```